stats/release_queue.php

```diff
@@ -152,23 +152,23 @@
     }
 
 
-    $comments_url1 = mysqli_real_escape_string(DPDatabase::get_connection(), "<a href='$code_url/project.php?id=");
-    $comments_url2 = mysqli_real_escape_string(DPDatabase::get_connection(), "'>");
-    $comments_url3 = mysqli_real_escape_string(DPDatabase::get_connection(), "</a>");
+    $comments_url1 = DPDatabase::escape("<a href='$code_url/project.php?id=");
```

Comment:
are these unused?

Reply:
They're used in the SQL statement directly below this. HTML in the database! :)

Reply:
Well, HTML in DB queries because of the way some of the tables are output. It's horrific, don't look at the code.